lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -272,66 +273,82 @@ def emit(group, stream, event, metadata)
     end
 
     def get_events(log_group_name, log_stream_name)
-      throttling_handler('get_log_events') do
-        request = {
-          log_group_name: log_group_name,
-          log_stream_name: log_stream_name
-        }
-        request.merge!(start_time: @start_time) if @start_time
-        request.merge!(end_time: @end_time) if @end_time
+      request = {
+        log_group_name: log_group_name,
+        log_stream_name: log_stream_name
+      }
+      request.merge!(start_time: @start_time) if @start_time
+      request.merge!(end_time: @end_time) if @end_time
+      
+      if @use_log_group_name_prefix
+        log_next_token = next_token(log_stream_name, log_group_name)
+      else
+        log_next_token = next_token(log_stream_name)
+      end
+      
+      request[:next_token] = log_next_token if !log_next_token.nil? && !log_next_token.empty?
+      request[:start_from_head] = true if read_from_head?(log_next_token)
+      
+      # Only apply throttling retry to the API call, not the whole method
```

Comment:
```suggestion
```

Information that can be found in the code is not required for comments.
(Maybe those comments are for PR. Thanks. I understand the code. Let's remove them now.)